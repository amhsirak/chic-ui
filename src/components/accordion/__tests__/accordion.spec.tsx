import Accordion from '..';
import { render } from '../../../test/utils';

/* eslint-disable */
describe('Accordian Component', () => {
    it('Renders Correctly', () => {
        const AccordionGroup = render(
            <>
                <Accordion header='Hello World' description='Short Description' />
            </>
        );
        expect(AccordionGroup).toMatchSnapshot();
    });
});
