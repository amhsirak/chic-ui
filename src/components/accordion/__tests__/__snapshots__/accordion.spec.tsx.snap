// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accordian Component Renders Correctly 1`] = `
<div
  className="sc-bdfBQB"
>
  <h4
    className="sc-gsTEea fnlNQt accordian-header"
    onClick={[Function]}
  >
    <div
      className="accordian-toggle-button"
    >
      <span />
    </div>
    Hello World
  </h4>
  <div
    className="sc-dlfnuX dEShIn desc-wrapper"
  >
    <div
      className="sc-hKgJUU juZUcl accordian-description close"
    >
      Short Description
    </div>
  </div>
</div>
`;
